tool/tsh/common/tsh_test.go

```diff
@@ -2542,18 +2542,15 @@ func TestSSHCommands(t *testing.T) {
 func tryCreateTrustedCluster(t *testing.T, authServer *auth.Server, trustedCluster types.TrustedCluster) {
 	ctx := context.TODO()
 	for i := 0; i < 10; i++ {
-		log.Debugf("Will create trusted cluster %v, attempt %v.", trustedCluster, i)
```

Comment:
These could use `t.Logf`. (I'm OK with removing too if you prefer to keep as-is.)